client/components/HeartBox.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import { faHeart } from '@fortawesome/free-solid-svg-icons'
+import BoxButton from './BoxButton';
+library.add(faHeart);
+
+
+class HeartBox extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <span style={{ display: this.props.isSelected ? "inline-block" : "none" }}>
```

Comment:
here it would be better to have the component be rendered conditionally as opposed to having it always rendered and not having it shown with css

Reply:
I will resolve this change in the next version due to a css sizing issue that was blocking me.  I will ask follow up in person on Tuesday.